docs/configuration/events/egg-events.rst

```diff
@@ -75,16 +75,22 @@ geofence            Geofence around the event.
 Time
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-================= =========================================================
-DTS               Description
-================= =========================================================
-hatch_time_left   Time remaining until the egg hatches.
-12h_hatch_time    Time when the egg will hatch, formatted in 12h.
-24h_hatch_time    Time when the egg will hatch, formatted in 24h.
-raid_time_left    Time remaining until the raid ends.
-12h_raid_end      Time when the raid ends, formatted in 12h.
-24h_raid_end      Time when the raid ends, formatted in 24h.
-================= =========================================================
+======================= ===============================================================
+DTS                     Description
+======================= ===============================================================
+hatch_time_left         Time remaining until the egg hatches.
+12h_hatch_time          Time when the egg will hatch, formatted in 12h.
+24h_hatch_time          Time when the egg will hatch, formatted in 24h.
+raid_time_left          Time remaining until the raid ends.
+12h_raid_end            Time when the raid ends, formatted in 12h.
+24h_raid_end            Time when the raid ends, formatted in 24h.
+hatch_time_left_minutes Time remaining until the egg hatches without seconds.
```

Comment:
Maybe we should update these more explicitly? (formatted h?:mm:ss) vs (formatting h?:mm)

Reply:
I'm thinking of adding an example column so that it's defined a little more explicitly